### PR TITLE
Update confirmation page pattern

### PR DESCRIPTION
**Pull request by timpaul**
- Add example image
- Slight rewording
- Link to example in prototyping kit

### CODE REVIEW

**Comment by joelanman** [comment]
not sure about the use of 'need' in this line:

`any actions the user needs to perform`

if they need to do something else, then they're not done, so they shouldn't see a confirmation page. Maybe this concept should be part of the guidance too?

suggest replacement: 'any related actions the user can perform'

**Comment by timpaul** [comment]
I think we should open this out as an area for more research. There's a grey area - where you've finished the immediate task but your overall goal isn't yet done. Perhaps we need a variant of this page that tones down the 'Success!' aspect.

**Comment by joelanman** [comment]
Note from real life discussion:

Publish that this pattern is for "you're done" - not "you have more to do". Call for research into the latter situation as we're not aware of significant research on it.

File: service-manual/user-centred-design/resources/patterns/confirmation-pages.md [hunk 1]
```diff
@@ -13,23 +13,44 @@ breadcrumbs:
...
-* their reference number if there is one
-* what further actions they need to perform, and when
-* what further actions they can expect from the service, and when
-* who they should contact with any queries or complaints
-* about any related information and services
+<h2 class="heading-36" id="section-1">Guidance</h2>
 
-Use this pattern to link to the [feedback page](/service-manual/user-centred-design/resources/patterns/feedback-pages) for the service:
+The page should include:
+
```
Comment (edwardhorsford) [inline service-manual/user-centred-design/resources/patterns/confirmation-pages.md]: Extra line break?

File: service-manual/user-centred-design/resources/patterns/confirmation-pages.md [hunk 2]
```diff
@@ -13,23 +13,44 @@ breadcrumbs:
...
+* a reference number (if there is one)
+* details of what happens next and when
+* contact details for the service
+* links to related information and services
+* a prompt to give [feedback](/service-manual/user-centred-design/resources/patterns/feedback-pages) on the service
+* a way of saving a record of the transaction (eg as a PDF)
+
```
Comment (edwardhorsford) [inline service-manual/user-centred-design/resources/patterns/confirmation-pages.md]: Extra line break

**Comment by edwardhorsford** [comment]
I think this is a good improvement. It leave room for further discussion about 'next steps', which feels appropriate.

Very often a service will have taken payment and the application submitted, but there's still one more step for the user - eg to send off documents. Right now this is the pattern services tend to use for that. For passports it worked ok for us, though I would like to improve it.

**Pull request merged**